lib/JITServer/JITServer.cpp

```diff
@@ -174,6 +163,7 @@ ServerInitializeThreadContext(
     }
     catch (Js::OutOfMemoryException)
     {
+        CloseHandle((HANDLE)threadContextData->processHandle);
```

Comment:
What about other types of exception ? Are we guarantied only OOM can happen here ?